src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -574,13 +574,14 @@ std::string ContextAttribute::renderAsJsonObject
         }
         else // regular number
         {
-          effectiveValue      = toString(numberValue);
-          valueIsNumberOrBool = true;
+          effectiveValue = toString(numberValue);
+          withoutQuotes  = true;
         }
         break;
 
       case ValueTypeNone:
         effectiveValue = "null";
+        withoutQuotes  = true;
```

Comment:
withoutQuotes is ok.
isString (the negation) would be ok as well (I guess).

NTC